core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2814,6 +2814,14 @@ secret_configs:
     - build_farm
     name: endurance-sa
     namespace: test-credentials
+- from:
+    sa.cluster-display.hive.config:
+      attachment: sa.cluster-display.hive.config
```

Comment:
How do we generate this config?

Reply:
For now, manually.
We have only 3 SA on hive: `ci/config-updater`, `ci/hive`, `ci-cluster-display`.

Reply:
Did not use our `generator` because we did not hook the config for the tool.
I promise I will do it if the fourth one comes (the fourth one is actually the 3rd because config-updater is manual work anyway).